src/scripts/validation.js

```diff
@@ -24,36 +23,65 @@ const hideInputError = (formElement, inputElement, config) => {
 
 };
 
-const hasInvalidInput = (inputList) => {
+export function checkInputValidity(formElement, inputElement, config) {
+  if (!inputElement.validity.valid) {
+    showInputError(formElement, inputElement, inputElement.validationMessage, config);
+  } else {
+    hideInputError(formElement, inputElement, config);
+  }
+};
+
+export const hasInvalidInput = (inputList) => {
   return inputList.some((input) => {
     return !input.validity.valid;
   });
 };
 
+export function disableButton(buttonElement, inactiveButtonClass) {
+  if(!buttonElement) return;
+  buttonElement.classList.add(inactiveButtonClass)
+  buttonElement.disabled = true;
+  console.log("Button should be disabled", buttonElement);
+}
+
 
-const toggleButtonState = (inputList, buttonElement, config) => {
+export const toggleButtonState = (inputList, buttonElement, config) => {
+  // console.log(hasInvalidInput(inputList));
   if (hasInvalidInput(inputList)) {
-   disableButton(buttonElement, config);
+    buttonElement.disabled = true;
+    buttonElement.classList.add(config.inactiveButtonClass);
   } else {
     buttonElement.disabled = false;
     buttonElement.classList.remove(config.inactiveButtonClass);
   }
 };
 
-const disableButton = (buttonElement, config) => {
-  buttonElement.disabled = true;
-  buttonElement.classList.add(config.inactiveButtonClass);
-}
+// export const disableButton = (buttonElement, config) => {
+//   if (buttonElement) {
+//     buttonElement.disabled = true;
+//     buttonElement.classList.add(config.inactiveButtonClass)
+//   }
+// }
 
-const resetValidation = (formElement, inputList, config) => {
+//   buttonElement.disabled = true;
+//   buttonElement.classList.add(config.inactiveButtonClass);
+// }
+
+export const resetValidation = (formElement, config) => {
+  const inputList = Array.from(formElement.querySelectorAll(config.inputSelector));
+  const buttonElement = formElement.querySelector(config.submitButtonSelector);
   inputList.forEach((input) => {
     hideInputError(formElement, input, config);
   });
+  // toggleButtonState(inputList, buttonElement, config);
+  disableButton(buttonElement, config.inactiveButtonClass);
+
+  formElement.reset();
```

Comment:
`formElement.reset();` is not needed in `resetValidation`, that's why the inputs of the `profile` form are empty on opening